xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/CommentsTab.java

```diff
@@ -55,11 +56,22 @@ public String getCurrentAuthor()
 
     public boolean isCommentFormShown()
     {
+        toggleComments();
         WebElement commentForm =
             getDriver().findElement(By.xpath("//form[@id='AddComment']/fieldset[@id='commentform']"));
         return commentForm.isDisplayed();
     }
 
+    private void toggleComments()
+    {
+        XWikiWebDriver driver = getDriver();
+        List<WebElement> openCommentForm = driver.findElementsById("OpenCommentForm");
```

Comment:
You don't need to rely on a list, you have a `findElementById`. You could even use `findElementWithoutWaiting(By)` there with a `By.id` since the button is supposed to be displayed.